SmartDeviceLink/SDLCarWindow.m

```diff
@@ -37,14 +39,24 @@ @interface SDLCarWindow ()
 
 @property (assign, nonatomic, getter=isVideoStreamStarted) BOOL videoStreamStarted;
 
+@property (assign, nonatomic) float sdl_scale;
```

Comment:
The scale needs to be updated in the car window class just like the touch manager class. Otherwise the scale is always going to be 1.0. When I tested with a scale bigger than 1.0 no video streamed because the dimensions were wrong.